### ISSUE TITLE
Export survey responses as .csv

### ISSUE THREAD

**Issue by trangmei** [Coursemology/coursemology2#3045]
Feature requested by Louie for CS3216 initial survey.

- [ ] To discuss with @kxmbrian on implementation.

**Comment by dexterleng** [comment]
I've looked at LimeSurvey (open source survey site). For this Coursemology survey:

![screen shot 2018-12-20 at 3 04 32 pm](https://user-images.githubusercontent.com/34204380/50269288-cb928d80-0468-11e9-9d71-59faae3ccbd5.png)

Would look like this when duplicated and exported on LimeSurvey:

```
"user_id","date_submitted","Do you like recursion? [One]","Do you like recursion? [Two]","Do you like recursion? [Three]","Do you like recursion? [Four]","What is your opinion on it?","Can all recursion algos be written iteratively?","2 + 2 = 2 + 2"

```

Observations:
1. Multiple Response Questions that have n responses would have n columns.
2. Empty responses are represented with `""` instead of just nothing.
3. MCQs responses are mapped to integers instead of the text answer (in the context of Coursemology it would be the `answer_id`.

**Comment by louietyj** [comment]
Fixed by #3305
